README.md

```diff
@@ -18,7 +18,7 @@ TRANSLATION_SERVICE_ACCOUNT_KEY="..."
 FACEBOOK_AUTH_TOKEN="..."
 
 FORTIS_FEATURE_SERVICE_HOST="..."
-FORTIS_SERVICES_APPINSIGHTS_KEY="..."
+APPINSIGHTS_INSTRUMENTATIONKEY="..."
```

Comment:
This is amazing. The AppInsights SDKs are inconsistent between each other. Java uses APPLICATION_INSIGHTS_IKEY and all the others use APPINSIGHTS_INSTRUMENTATIONKEY. Given that the Java SDK's key can't be configured, let's be consistent and use its env var name everywhere.